packages/remix/src/ssr/rootAuthLoader.ts

```diff
@@ -49,7 +49,8 @@ export const rootAuthLoader: RootAuthLoader = async (
 
   const loadedOptions = loadOptions(args, opts);
   // Note: authenticateRequest() will throw a redirect if the auth state is determined to be handshake
-  const requestState = await authenticateRequest(args, loadedOptions);
+  const _requestState = await authenticateRequest(args, loadedOptions);
+  const requestState = { ...loadedOptions, ..._requestState };
```

Comment:
Should @clerk/backend be aware of these new redirect urls ? Seems like `afterXUrl` are being returned as part of the SignedIn/OutState

Reply:
❓ Aren't those properties only available in the client? I believe that the only reason the `afterXUrl` exist in the backend is for debugging.
@nikosdouvlis Do you have any more context?

Reply:
Yes, i would expect them to only be used in the client as well.
Do you think we should update add the new urls in the request state for usage in debugging as well ? Or is it ok to leave the PR as it is now ?

Reply:
Why wouldn't we want them in `loadedOptions`? I think we can safely add them - improved debuggability is a welcome side effect